### PR TITLE
#170947373 Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
Install and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client

#### Description of Task to be completed?

- [x] Install style loaders
- [x] Updated webpack.config.js

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/170947373

#### Screenshots (if appropriate)
Screen Shot 2020-04-16 at 14 20 34

[finishes #170947373]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +26,18 @@
...
     "eslint-config-airbnb": "^18.1.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^2.5.1",
     "html-webpack-plugin": "^3.2.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^4.13.1",
+    "sass-loader": "^8.0.2",
+    "style-loader": "^1.1.4",
```
Comment (AbdoulNuru) [inline package.json:46]: @blaise82 just curious, among those packages above don't we need saas package too?

File: src/styles/styles.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+
+.center {
+    color: $example-color;
+    text-align: center;
+}
```
Comment (hakoemmy) [inline src/styles/styles.scss]:
```suggestion
}

```
New line

File: src/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+$example-color: #e2e2e2;
```
Comment (hakoemmy) [inline src/styles/variables.scss]:
```suggestion
$example-color: #e2e2e2;

```

File: package.json (hunk 1 above)
Comment (blaise82) [reply package.json:46]: which sass pachage?

Comment (blaise82) [reply package.json:46]: - node-sass provides binding for Node.js to LibSass, a Sass compiler.
- sass-loader is a loader for Webpack for compiling SCSS/Sass files.
- style-loader injects our styles into our DOM.
- css-loader interprets @import and @url() and resolves them.
- mini-css-extract-plugin extracts our CSS out of the JavaScript bundle into a separate file, essential for production builds.

**Review by AbdoulNuru — APPROVED** [review]

**Review by hakoemmy — APPROVED** [review]

**Review by Moise1 — APPROVED** [review]

**Pull request merged**